Vostok.Metrics.System/Host/HostMetrics.cs

```diff
@@ -61,13 +71,28 @@ public class HostMetrics
         /// <summary>
         /// Disk space info per volume.
         /// </summary>
-        public Dictionary<string, DiskSpaceInfo> DiskSpaceInfos { get; set; }
+        public Dictionary<string, DiskSpaceInfo> DisksSpaceInfo { get; set; }
+
+        /// <summary>
+        /// Disk usage info per device.
+        /// </summary>
+        public Dictionary<string, DiskUsageInfo> DisksUsageInfo { get; set; }
 
         /// <summary>
         /// TCP connections per state.
         /// </summary>
         public Dictionary<TcpState, int> TcpStates { get; set; }
 
+        /// <summary>
+        /// Network sent bytes per second (across all interfaces).
+        /// </summary>
+        public double NetworkSentBytesPerSecond { get; set; }
```

Comment:
A quick idea: could we also compute a network utilization metric by comparing these figures with interface speed properties?

Reply:
Is it really that useful? 
Information for LAN interfaces can be gathered from `sys/class/net/<ethName>/speed`.
Information for Wireless interfaces can be gathered from `proc/net/wireless` (Not maximum speed, see details below).
There are a few problems:
1) LAN interfaces almost never reach their maximum speed, because it requires some special conditions. (I'm not really sure if that's the case, but I think that according to this fact the metric will not be very useful)
2) Wireless interfaces maximum speed can't be calculated correctly (At least I didn't find any useful information about this). 
3) Retrieving needed information is not very convenient. In docker we have 3 interfaces + lo interface, and therefore we have to "hardcode" docker filter.
4) There are some interfaces with unobtainable speed property. It is not clear, whether we should NOT calculate maximum network speed at all (because we do not have enough info), or not. If we shouldn't, then what do we need to return as a metric value? 

I haven't investigated how to find these values on Windows yet, but I think I will face similar problems.

Reply:
A couple of quick comments here:
- We're not interested in wireless interfaces at all;
- The statement about network interfaces almost never reaching their capacity is simply not true in our environment. We have a variety of 1G and 10G adapters and a number of services that could easily utilize the links under heavy load

Reply:
Ok, then I assume that even if there are wireless interfaces on host, their potential speed (and utilization as well) is much slower in comparison to LAN interfaces. (At least in our case)

So I will not remove them from previous network metrics and I will not count them in total max network speed. 
(It would probably be better to remove them, but then we will have to find another way to get TCP State per network interface in order to filter them afterwards).